pandas/io/sql.py

```diff
@@ -180,6 +178,15 @@ def execute(sql, con, params=None):
     -------
     Results Iterable
     """
+    warnings.warn(
+        "pandas.io.sql.execute will be removed in pandas 3.0",
+        DeprecationWarning,
```

Comment:
We want to raise a FutureWarning here.

Could you remove the deprecation for now? We want to finish enforcing old deprecations before we add new ones. Would be better to separate this from the other changes here

Reply:
Ok, do you want me to leave it in the user guide, but just change it so that it doesn't imply you can pass an Engine?  I've updated the PR this way, and removed the warning (or did you want me to change the warning to FutureWarning in this PR?)

Reply:
No removing the warning is fine, the FutureWarning comment was for the future.

I am ok with deleting it from the user guide immediately

Reply:
Ok, it is removed from the user guide now.